packages/braintree-integration/src/braintree-fastlane/braintree-fastlane-payment-initialize-options.ts

```diff
@@ -70,6 +73,11 @@ export default interface BraintreeFastlanePaymentInitializeOptions {
      * no matter what strategy was initialised first
      */
     styles?: BraintreeFastlaneStylesOption;
+    /**
+     * 3DS
+     */
+    threeDSecure?: BraintreeThreeDSecureOptions;
```

Comment:
Do you still need this option here? If not please remove it and `BraintreeThreeDSecureOptions` interface

Reply:
removed